frontend/integration-tests/tests/devconsole/app-launcher.scenario.ts

```diff
@@ -0,0 +1,44 @@
+import { browser, $ } from "protractor";
+import { appHost, checkLogs, checkErrors } from "../../protractor.conf";
+import {
+  switchPerspective,
+  Perspective,
+} from "../../views/devconsole-view/app-launcher.view";
+
+describe("Application Launcher Menu", () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/k8s/cluster/projects`);
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  async function dev() {
+    await switchPerspective(Perspective.Developer);
+  }
+
+  async function admin() {
+    await switchPerspective(Perspective.Administrator);
+  }
+
+  var pageSidebar = $('#page-sidebar .pf-c-nav .pf-c-nav__list');
+
+
+  it("Switch to dev perspective", async () => {
+    await admin();
+    await switchPerspective(Perspective.Developer);
+    expect(pageSidebar.getText()).toContain('Topology');
```

Comment:
1. What about checking the text of `#page-sidebar .oc-nav-header h1`?
2. You should also check that the content of page changed.

Reply:
1. Yeah I can check header also.
2. To check the change in the content of the page I'm using this expect.

Reply:
2) this expect is checking the sidebar, not the page itself, i.e. what is inside element `div#content`